docs/modules/operation/pages/quick-start/performance-data.adoc

```diff
@@ -0,0 +1,41 @@
+
+= Step 4: Set Up Performance Data Collection
+
+This section describes how to set up performance data collection in {page-component-title}.
+
+{page-component-title} collects performance data using xref:reference:daemons/daemon-config-files/collectd.adoc[collectd], which is enabled by default.
```

Comment:
Confusingly, there is a totally unrelated open-source project called [`collectd`](https://collectd.org/) which does the same sorts of things and which many of our new users may have encountered in the past. Let's add an info box calling out the distinction, please.